Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>.pytest_cache\r\n.idea\r\n.gitignore
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
--- a/.gitignore	(revision 7c79c2212b8cb3618e05b5c2cd553abe4c58fbf9)
+++ b/.gitignore	(date 1659841759319)
@@ -1,3 +1,16 @@
 .pytest_cache
 .idea
-.gitignore
\ No newline at end of file
+log
+report
+temp
+*.pyc
+*.log
+*.html
+*.json
+*.js
+*.css
+.gitignore
+*.iml
+__pycache__
+
+
Index: .idea/ddddpytest.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<module type=\"PYTHON_MODULE\" version=\"4\">\r\n  <component name=\"NewModuleRootManager\">\r\n    <content url=\"file://$MODULE_DIR$\">\r\n      <excludeFolder url=\"file://$MODULE_DIR$/.pytest_cache/v\" />\r\n      <excludeFolder url=\"file://$MODULE_DIR$/.pytest_cache\" />\r\n    </content>\r\n    <orderEntry type=\"inheritedJdk\" />\r\n    <orderEntry type=\"sourceFolder\" forTests=\"false\" />\r\n  </component>\r\n  <component name=\"PyDocumentationSettings\">\r\n    <option name=\"format\" value=\"PLAIN\" />\r\n    <option name=\"myDocStringFormat\" value=\"Plain\" />\r\n  </component>\r\n  <component name=\"TestRunnerService\">\r\n    <option name=\"PROJECT_TEST_RUNNER\" value=\"py.test\" />\r\n  </component>\r\n</module>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/ddddpytest.iml b/.idea/ddddpytest.iml
--- a/.idea/ddddpytest.iml	(revision 7c79c2212b8cb3618e05b5c2cd553abe4c58fbf9)
+++ b/.idea/ddddpytest.iml	(date 1659841847659)
@@ -5,7 +5,7 @@
       <excludeFolder url="file://$MODULE_DIR$/.pytest_cache/v" />
       <excludeFolder url="file://$MODULE_DIR$/.pytest_cache" />
     </content>
-    <orderEntry type="inheritedJdk" />
+    <orderEntry type="jdk" jdkName="Python 3.10" jdkType="Python SDK" />
     <orderEntry type="sourceFolder" forTests="false" />
   </component>
   <component name="PyDocumentationSettings">
